agent/connect/ca/provider_vault_test.go

```diff
@@ -193,24 +212,35 @@ func TestVaultCAProvider_Configure(t *testing.T) {
 }
 
 func TestVaultCAProvider_SecondaryActiveIntermediate(t *testing.T) {
-
 	SkipIfVaultNotPresent(t)
 
-	provider, testVault := testVaultProviderWithConfig(t, false, nil)
-	defer testVault.Stop()
+	t.Parallel()
```

Comment:
Are these safe to run parallel now?

Reply:
They always were I believe. Each test gets its own vault process.

I added this b/c it shaved the runtime down by a factor of ~5. Locally it was ~28s -> ~6s.